File: 0001_0100/0068/0068.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n;
	char s[7];

	fscanf(f, "%s%d", s, &n);
	fprintf(q, n % 2 || *s == 'H' ? "Yes" : "No" );
	return 0;
}

File: 0001_0100/0068/0068.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s string
	var n int
	fmt.Fscan(f, &s, &n)
	r := "No"
	if s == "Home" || n%2 == 1 {
		r = "Yes"
	}
	fmt.Fprint(q, r)
}
